source/javascripts/services/logger.ts

```diff
@@ -1,41 +1,89 @@
-import { datadogLogs } from "@datadog/browser-logs";
+import { pick } from "underscore";
+import { datadogLogs, StatusType, HandlerType, Logger as DLogger } from "@datadog/browser-logs";
 import { Context } from "@datadog/browser-core";
+import { getAppSlug } from "./app-service";
 import { WFEWindow } from "../typings/global";
 
+const w = window as WFEWindow;
```

Comment:
Good idea, we can re-declare `window` type
```suggestion
declare var window: WFEWindow;
```